packages/node/src/behaviors/on-off/OnOffServer.ts

```diff
@@ -9,24 +9,24 @@ import { GeneralDiagnostics } from "#clusters/general-diagnostics";
 import { OnOff } from "#clusters/on-off";
 import { RootEndpoint } from "#endpoints/root";
 import { MaybePromise, Time, Timer } from "#general";
+import { ClusterType } from "#types";
 import { OnOffBehavior } from "./OnOffBehavior.js";
 
-const Base = OnOffBehavior.with(OnOff.Feature.Lighting);
+const OnOffLogicBase = OnOffBehavior.with(OnOff.Feature.Lighting);
 
 /**
  * This is the default server implementation of {@link OnOffBehavior}.
  *
- * This implementation includes all features of {@link OnOff.Cluster} and automatically enables the "Level Control
- * for Lighting" Feature. You should use {@link OnOffServer.with} to specialize the class for the features your
- * implementation supports. Alternatively you can extend this class and override the methods you need to change or add
- * mandatory commands.
+ * This implementation includes all features of {@link OnOff.Cluster}. You should use {@link OnOffServer.with} to
+ * specialize the class for the features your implementation supports. Alternatively you can extend this class and
+ * override the methods you need to change or add mandatory commands.
  *
- * The "OffOnly" feature is automatically supported because the commands are disabled by conformance.
+ * The "OffOnly" and "Lighting" features are automatically supported because the commands are disabled by conformance.
  * The default implementation do not contain any logic for the DeadFrontBehavior feature because this is very use case
  * specific, so this needs to be implemented by the device implementor as needed.
  */
-export class OnOffServer extends Base {
-    declare protected internal: OnOffServer.Internal;
+export class OnOffServerLogic extends OnOffLogicBase {
```

Comment:
The "Logic" suffix feels a little weird to me, though I know we use it in other clusters... What if we use the convention "CompleteOnOffServer", "OnOffBaseServer", "OnOffWithAllServer" or "FullFeaturedOnOffServer"?

They're all a little contorted but the "Logic" suffix makes it seem like the class is something other than a server.

Reply:
Good topic. I will change that in a separate PR for all where we do it like that